scripts/bash_completion/optirun

```diff
@@ -54,7 +54,8 @@ _optirun() {
           *)
```

Comment:
Completely agree, but the amount of options is huge, as shown in this [link](http://www.virtualgl.org/vgldoc/2_1/#hd0019001), so starting point is to at least autocomplete up to --vgl-options. So that is why I included it.